static/js/tools/choropleth/choropleth.tsx

```diff
@@ -382,7 +385,9 @@ function determineColorPalette(dict, pc: boolean, popMap: []): number[] {
 function formatGeoValue(geoValue, isPerCapita) {
   // Per capita values may be difficult to read so add a per count field.
   // E.g. 0.0052 or 5.2 per 1000.
-  if (isPerCapita || geoValue === "No Value") {
+  if (!isPerCapita || geoValue === "No Value") {
```

Comment:
!isPerCapita && !(geoValue === "No Value")

Reply:
The old code was actually incorrect. The current case code is implemented correctly.